build/bin/smoke-onos-config

```diff
@@ -15,6 +15,13 @@
 
 # This script is intended to be run from the onos-test directory as part of CI
 
+set -eu -o pipefail
+
+GREEN="\e[32m"
```

Comment:
There are 14 smoke tests in total. This one will now behave differently than the others. If we want to go this way we should make the corresponding changes to all of them

Also - how will this effect what Jenkins logs? Most of the time these tests are not run on a terminal.

Reply:
Ah good point, I think this would mess up the Jenkins log, let me remove it